File: app/src/main/java/com/example/android/test1/APIService.java
package com.example.android.test1;

import com.example.android.test1.POJO.Weather;

import retrofit2.Call;
import retrofit2.http.POST;
import retrofit2.http.Query;

public interface APIService {
    @POST("current.json")
    Call<Weather> getCurrentCityWeather(@Query("key") String key,
                                 @Query("q") String city);

    @POST("forecast.json")
    Call<Weather> getForecastCityWeather(@Query("key") String key,
                                        @Query("q") String city,
                                         @Query("days") int days);
}

File: app/src/main/java/com/example/android/test1/Adapters/APIService.kt
package com.example.android.test1.Adapters

import com.example.android.test1.data.Weather
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface APIService {

    @GET("/current")
    fun getCurrentCityWeather(
        @Query("access_key") key: String,
        @Query("query") city: String
    ): Call<Weather>

    @GET("/forecast")
    fun getForecastCityWeather(
        @Query("access_key") key: String,
        @Query("query") city: String
    ): Call<Weather>
}